src/Console/Command/FixCommand.php

```diff
@@ -178,10 +179,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
         if ('none' === $progressType || null === $stdErr) {
             $progressOutput = new NullOutput();
         } elseif ('run-in' === $progressType) {
```

Comment:
as already described in one of outdated comments, i don't see having that iterating as an issue.
for small collection of files - it's fast.
for extra big collection of files - it won't be that slower, and not having % how far you are when you fix 10_000 files is not nice